app/src/main/java/kth/jjve/xfran/EditEventActivity.java

```diff
@@ -36,21 +42,23 @@ protected void onCreate(Bundle savedInstanceState) {
         eventStartTimeEdit = findViewById(R.id.eventStartTimeEdit);
         eventEndTimeEdit = findViewById(R.id.eventEndTimeEdit);
         Button buttonSave = findViewById(R.id.eventSave);
+        Button buttonClose = findViewById(R.id.close);
+        Button buttonExport = findViewById(R.id.savetogoogle);
 
         String s_date = "Date: " + CalendarUtils.cleanDate(CalendarUtils.selectedDate);
         eventDate.setText(s_date);
 
         /*-------- LISTENERS ------------*/
-        buttonSave.setOnClickListener(v -> {
-            saveEvent();
-        });
+        buttonClose.setOnClickListener(v -> { finish(); });
+        buttonSave.setOnClickListener(v -> { saveEvent(); });
+        buttonExport.setOnClickListener(v -> { saveEventToCalendar(); });
     }
 
-    private void saveEvent(){
-        //method to save the name, start and end time of an event into a list
-        String s_eventName = eventName.getText().toString(); //given name
-        String startTime = eventStartTimeEdit.getText().toString(); //start time
-        String endTime = eventEndTimeEdit.getText().toString(); //end time
+    private void setEventInApp(){
+        // method to obtain the events name and start/end time
+        s_eventName = eventName.getText().toString(); //given name
```

Comment:
The comments at the end of the line are actually not needed :)
It can be read from the code itself